.github/workflows/release.yml

```
@@ -0,0 +1,36 @@
+name: Add zip file to S3 bucket
+
+on: [release]
+
+jobs:
+  create-zip:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: vimtor/action-zip@v1
+        with:
+          files: ./
+          dest: filename.zip
+
+      - uses: actions/upload-artifact@v1
+        with:
+          name: zip-repository
+          path: ${{ github.workspace }}/filename.zip
+
+
+  send-zip:
+    runs-on: ubuntu-latest
+    needs: create-zip
+    steps:
+      - name: Download artifact
+        uses: actions/download-artifact@v3
+        with:
+          name: zip-repository
```

```suggestion
          name: PWA
```
**chore**: Update name if updating above.